src/w3c/seo.js

```diff
@@ -50,5 +50,11 @@ export function run(conf, doc, cb) {
     linkElem.setAttribute("href", conf.canonicalURI);
     doc.head.appendChild(linkElem);
   }
+  var metaDescription = doc.querySelector("#abstract p:first-of-type")
```

Comment:
As this is universally useful, we should make a new module ("core/seo").  Also, remember to only use "var" for global variables (basically never): always use `const` unless the variable is going to change , in which case you use "let". 

```JS
export async function run(conf, doc, cb) {
  const metaDescription = doc.querySelector("#abstract p:first-of-type");
  if (!metaDescription) {
    return; // no abstract, so abort early
  }
  // This is not critical, so let's continue other processing first:
  cb();
  await doc.respecIsReady;
  const metaElem = doc.createElement("meta");
  metaElem.name = "description";
  metaElem.content = metaDescription;
  doc.head.appendChild(metaElem);
}
```